kuronometer-core/src/main/scala/com/github/pedrovgs/kuronometer/free/domain/Reporter.scala

```diff
@@ -0,0 +1,8 @@
+package com.github.pedrovgs.kuronometer.free.domain
+
+sealed trait Report
```

Comment:
Most of the time you want to express these as `sealed trait A extends Product with Serializable` http://stackoverflow.com/questions/36525804/scala-case-class-extending-product-with-serializable it's one of those odd scala things :(